configuration/items.md

```diff
@@ -297,7 +297,7 @@ In the example below, the entry `MAP(window_esp.map)` causes the output of the `
 Contact Livingroom_Window "Ventana del salón [MAP(window_esp.map):%s]"
 ```
 
-Please refer to the article on [Transformations](transform.html) for more usage details and a list of available transformation services.
+Please refer to the article on [Transformations](/docs/configuration/transformations.html) for more usage details and a list of available transformation services.
```

Comment:
Omitting the full path here would have been sufficient because you're already in the `/docs/configuration` "folder".
(Just mentioning it, both are fine.)

Reply:
Thx for mentioning.
I am a friend of full path^^